File: Chapter_2/anthropic/nursery.py
import os
from anthropic import Anthropic
from dotenv import load_dotenv

load_dotenv()

client = Anthropic(
    api_key=os.getenv("ANTHROPIC_API_KEY")
)

response = client.messages.create(
    model="claude-3-5-sonnet-20241022",
    max_tokens=100,
    messages=[{"role": "user", "content": "Twinkle, Twinkle, Little"}]
)

print(response.content[0].text)

File: essentials/chapter_3/anthropic/nursery.ts
import Anthropic from '@anthropic-ai/sdk';
import "dotenv/config";

const anthropic = new Anthropic({
  apiKey: process.env.ANTHROPIC_API_KEY,
});

const response = await anthropic.messages.create({
  model: "claude-sonnet-4-6",
  max_tokens: 100,
  messages: [{ role: "user", content: "Twinkle, Twinkle, Little" }],
});

const content = response.content[0];
if (content.type === "text") {
  console.log(content.text);
}
